pages/0_cateringpage.php

```diff
@@ -0,0 +1,43 @@
+<?php
+ setTitle("Catering pagina");
+
+ $text = base_query("SELECT * FROM setting WHERE name = 'CateringText' ")->fetch();
+?>
+
+<div>
```

Comment:
Doet deze div ook nog iets?

Reply:
in deze div staat alles samen, ook de text die geprint moet worden onder de carousel met de foto's